### DISCUSSION TITLE
Illa is crushing when you try to edit (expend the field) a date /date time box.

### DISCUSSION THREAD

**Discussion by dianaspetcu** [illacloud/illa-builder#2030] [English Support]
Can you fix this problem about the edit field (expend it) the date/date time boxes?

After I drag and drop the field, when I try to expend the field, the app is blocked/freezing and ask me if I am waiting or close the app.
It is very annoying.
![image](https://github.com/illacloud/illa-builder/assets/132670969/7a22214c-e7dd-4605-bbb0-05051e6b4b3c)

**Comment by AruSeito** [comment]
Hello, we have attempted to optimize this issue.Can you try it again?
